api/subscriptions/pending_tx_test.go

```diff
@@ -146,7 +146,7 @@ func createTx(repo *chain.Repository, addressNumber uint) *tx.Transaction {
 	cla := tx.NewClause(&addr).WithValue(big.NewInt(10000))
 
 	return tx.MustSign(
-		new(tx.Builder).
+		new(tx.LegacyBuilder).
```

Comment:
probably we should check if dyntx follow same pending logic

Reply:
yes, that's something about the Transaction entity itself, the dynFeeTx only differs from the content, but I'll change one of the two tx to be of the new type